plugins/modules/zos_job_query.py

```diff
@@ -77,7 +77,7 @@
   type: bool
 jobs:
   description:
-     The list of z/OS job(s) and status.
+     The list of z/OS job(s) and status, or a single NOTFOUND record.
```

Comment:
Similar to other comment maybe slightly more verbose

The list of the z/OS job(s) and status  or a single NOTFOUND record when the job has not been run.

Reply:
@richp405 - seems that this one has been missed yet the other one is fixed
https://github.com/ansible-collections/ibm_zos_core/pull/260#discussion_r629592125